docs/FAQ.md

```diff
@@ -33,6 +33,32 @@ revision visible again.
 
 See [revsets] and [templates] for further guidance.
 
+### `jj` is said to record the working after `jj log` and every other command. Where can I see these automatic "saves"?  
+
+Indeed, every `jj` command updates the current "working-copy" revision, marked 
+with `@` in `jj log`. You can notice this by how the commit id of the
+working copy revision changes when it's updated. Note that, unless you move to
+another revision (with `jj new` or `jj edit`, for example), the change id will 
+not change.
+
+If you expected to see a historical view of your working-copy changes as an 
```

Comment:
I like this! I'd make a minor change:

> ... of your working-copy changes in `jj log`, as a chain of commits in a parent-child relationship, this is not the case.

One thing to note about it is that it's "a chain", not "an chain".

Reply:
thanks for the correction, should be fixed now.